### ISSUE TITLE
Assets scraping

### ISSUE THREAD

**Issue by RayLuxembourg** [LartTyler/MHWDB-Docs#6]
Is it possible add the assets when scrapping as well?
Most of the data on wiki comes with relevant assets.
If you need help with service like Amazon S3 let me know.

**Comment by LartTyler** [comment]
I actually started work on that the other day, I just need to wrap a few things up and then it's good push live. Keep an eye on the releases page!

**Comment by LartTyler** [comment]
I just realized this was the wrong issue that I responded to the other day. I didn't mean to close it, my bad.

@RayLuxembourg I'd like to add assets, but I'm not sure where I can scrape that information from. Any idea?

**Comment by cheng1994** [comment]
@LartTyler if you are scraping kiranico every item should contain image assets. Example: https://mhworld.kiranico.com/armor/leather-headgear. They appear to be showing them in a div with a css background url. 
`<div class="d-flex mr-3 img-thumbnail" style="width: 96px; height: 96px; background: url(&quot;https://mhworld.kiranico.com/images/armor_m.png&quot;) 0px 0px no-repeat;"></div>`
Scraping the url would be sufficient I would think. In my use case I would be able to work with just a scrapped url.

**Comment by RayLuxembourg** [comment]
@LartTyler 
I found this for icons - http://monsterhunter.wikia.com/wiki/MHW:_Lance_Weapon_Tree
icons for Element and Status Effect
icons for weapons

some more assets http://monsterhunter.wikia.com/wiki/Category:MHW_Images

**Comment by LartTyler** [comment]
Preview images for armor is available in [v1.6.0](https://github.com/LartTyler/MHWDB-Docs/releases/tag/1.6.0). There are a few issues with the data, caused by Kiranico having incomplete or incorrect information on the site, but it's stuff that I'm sure will be resolved over time. Weapon previews are next, followed by weapon icons.

**Comment by cheng1994** [comment]
@LartTyler Thanks man! Great job!

**Comment by kclay** [comment]
@LartTyler I know this is already completed and great work but may want to look into this site https://monsterhunterworld.wiki.fextralife.com/Alloy+Armor+Set

Most of their images follow a pretty common naming convention, I actually was going to take your api and make a script to map images for weapons,armor and skills. Now they do have some odd cases where they use a base image for say all 3 levels of a weapon but that can be easily resolved.

**Comment by LartTyler** [comment]
@kclay I don't actually consider this finished yet, I've got plans to add weapon images next. I didn't realize that fextralife had the images with transparent backgrounds though, so I may switch the API over to pull images from there instead. Thanks for pointing that out!

**Comment by baaronp7** [comment]
Yes transparent bg images would be amazing!

**Comment by kclay** [comment]
@LartTyler I created this script for my personal use. The purpose of it was to restructure the data models for my needs( restructure slots and sharpness,element models).
https://gist.github.com/kclay/a9216345c4c78edcb3561e46a8d2fe80

The `images` function handles all the cases I could find for getting images from fextralife. It handles looking for both male and female images as well as the base preview image they have on the site.

For weapons it resolves the preview image and tries to find the most relevent image for each weapon variant (Bone Kuri 1-3 for example)

Not sure if it would be possible to redo the datamodel a bit (you can see the example output by changing the file path) if thats possible (and I can open another issue) that would be perfect then I could use your api directly for searching rather then having to do it in memory.

Anyways maybe this helps out a bit. Great work by the way.

**Comment by LartTyler** [comment]
@kclay If there are other structure changes you'd like to discuss (aside from what we were talking about earlier in #13), feel free to open another issue. I can't promise the changes will happen, but I'd definitely be interested in hearing some ideas.

As far as assets are concerned, I just finished writing a scraper for the fextralife wiki for #15, so once I've got my backlog knocked down some, I'm planning on finishing up assets for as many objects as possible.

**Comment by RayLuxembourg** [comment]
@LartTyler Just in case you haven't seen those amazing assets.
https://monsterhunterworld.wiki.fextralife.com/Armor
https://monsterhunterworld.wiki.fextralife.com/Chest+Armor

they have proper transparent assets that can be displayed in any nice view.
Can't wait to have those assets in the app.

**Comment by LartTyler** [comment]
@RayLuxembourg Yep, I saw them. I'll be pulling those into the API once I knock my backlog down some, as well as using fextralife for weapon assets in the future.
